core/src/main/java/io/github/belgif/rest/guide/validator/output/model/ViolationFileObject.java

```diff
@@ -0,0 +1,11 @@
+package io.github.belgif.rest.guide.validator.output.model;
+
+public record ViolationFileObject(
+        String ruleId,
+        String description,
+        String message,
+        String type,
```

Comment:
"type" is kinda vague  (even though already used quite a time). Maybe rename to "level"